pkg/config/types.go

```diff
@@ -59,4 +59,8 @@ type Config interface {
 	ConfigFileUsed() string
 
 	BindPFlag(key string, flag *pflag.Flag) error
+
+	// Datadog specific
```

Comment:
I'd say better write actual documentation instead of this. "Datadog specific" is not helping understand this better. Are the rest not Datadog specific? Or, create an embedded interface which is called something else, and document that as being Datadog specific, saying what each method does, if you think this section will grow.

Reply:
you're right, the comment doesn't mean anything as it is, I'll fix this